src/http-client.ts

```diff
@@ -51,7 +51,20 @@ export class HttpClient {
     try {
       return await response.send(opts.data); // trigger request
     } catch (err) {
-      throw this.formatRequestError(err);
+      const res = err.response;
+
+      // Potentially contains PII information - Do not send to Sumo, just log to console.
+      throw {
```

Comment:
By just throwing a newly constructed object here, will it still be `instanceof Error`? It may be worth it to just throw the error as is. But it has been a minute this I've messed with superagent errors. Thoughts?

Reply:
You're right, I'm going to update this real quick and add the `Error` object to this so that there will be a stack trace.

So it'll be:
```
throw {
someStuff: res.someStuff,
error: new Error()
}
```

Reply:
Actually nevermind... we already have a stack trace in the object.

Reply:
I like throwing the object because its cleaner to read in the console and this is how it was logged before the Great Log Purge of 2021.